mmseg/models/decode_heads/dnl_head.py

```diff
@@ -26,8 +26,9 @@ def embedded_gaussian(self, theta_x, phi_x):
         pairwise_weight = torch.matmul(theta_x, phi_x)
         if self.use_scale:
             # theta_x.shape[-1] is `self.inter_channels`
-            pairwise_weight /= theta_x.shape[-1]**0.5
-        pairwise_weight /= self.temperature
+            pairwise_weight /= torch.tensor(
```

Comment:
may need to pass `device` argument and put the created tensors on the  same device as `pairwise_weight`